components/cdc/src/endpoint.rs

```diff
@@ -390,6 +407,8 @@ impl<T: 'static + RaftStoreRouter<RocksSnapshot>> Endpoint<T> {
             is_new_delegate = true;
             d
         });
+        let txn_extra_op = request.get_extra_op();
+        delegate.txn_extra_op = txn_extra_op;
```

Comment:
Can it work when a region is registered twice, and the two requests sets different option of `extra_op`?

Reply:
My mistake, we should check if `request.get_extra_op()` returns `TxnExtraOp::Noop` here.